packages/pilot/src/containers/Filter/optionsGroup/index.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import {
+  CardContent,
+  CardSection,
+  Row,
+  Col,
+  CheckboxGroup,
+} from 'former-kit'
+
+import style from './style.css'
+
+const optionsGroup = ({
+  disabled,
+  isOptionsEmptyOrCollapsed,
+  options,
+  selectedFilters,
+}) => {
+  if (isOptionsEmptyOrCollapsed) {
+    return null
+  }
+
+  return (
+    <CardContent>
+      <CardSection>
+        <CardContent>
+          <fieldset name="filters">
+            <Row flex>
+              {options.map(({ items, key, name }) => (
+                <Col key={name}>
+                  <div className={style.filtersTitle}>
+                    {name}
+                  </div>
+                  <CheckboxGroup
+                    columns={
+                      items.length > 6
```

Comment:
Apesar de simples esta lógica poderia estar em uma função separada fora do componente:
```suggestion
         getCheckboxGroupColumns(items.length)
```

Reply:
Feito!